beacon-chain/execution/engine_client.go

```diff
@@ -93,13 +99,28 @@ func (s *Service) NewPayload(ctx context.Context, payload interfaces.ExecutionDa
 	ctx, cancel := context.WithDeadline(ctx, d)
 	defer cancel()
 	result := &pb.PayloadStatus{}
-	payloadPb, ok := payload.Proto().(*pb.ExecutionPayload)
-	if !ok {
-		return nil, errors.New("execution data must be an execution payload")
-	}
-	err := s.rpcClient.CallContext(ctx, result, NewPayloadMethod, payloadPb)
-	if err != nil {
-		return nil, handleRPCError(err)
+
+	switch payload.Proto().(type) {
```

Comment:
Shouldn't this account for the EL?  To my understanding, the Capella execution payload is not accepted by Geth.

Getting constant warning that seems to cause the beacon sync to retreat.
```
Served engine_newPayloadV2               conn=127.0.0.1:39006 reqid=9 duration="17.91µs"   err="the method engine_newPayloadV2 does not exist/is not available"
```

And on Prysm side:
```
level=warning msg="Skip processing batched blocks" error="could not process block in batch: method not found: received an undefined execution engine error" prefix=initial-sync
```
I 'll be happy to open an issue if you need more info!

Network: Goerli-Prater
Geth: v1.10.26
Prysm: v3.2

Reply:
Which network is this ? if this is goerli, you will need to update to the latest version for each client

Reply:
I am on Goerli and just saw the [prysm release notes](https://github.com/prysmaticlabs/prysm/releases/tag/v3.2.2) about it. 

And on the [geth](https://github.com/ethereum/go-ethereum/releases/tag/v1.11.3) side as well.

Thanks for the quick response! 🙏  Resolving comment.